### ISSUE TITLE
Driver has invalid signature

### ISSUE THREAD

**Issue by jjonesnook** [electric-monk/360Controller#3]
I just installed this driver on my Yosemite MacBook Pro and I am having trouble getting the controller I have to work. I noticed this in the console logs and think this may be an issue:

com.apple.kextd[41]: ERROR: invalid signature for com.mice.driver.Xbox360Controller, will not load

**Comment by mele113** [comment]
Are you running RodrigoCard  0.15 version https://github.com/d235j/360Controller
its signed

**Comment by jjonesnook** [comment]
NO, I was running the version that came from the tattiboogle site itself. I didn’t even know about this version. I will give this a try later on.

Thanks,
Joe

From: "Patrick J. Mele"
Reply-To: electric-monk/360Controller
Date: Friday, May 15, 2015 at 9:31 AM
To: electric-monk/360Controller
Cc: Joseph Jones
Subject: Re: [360Controller] Driver has invalid signature (#3)

Are you running RodrigoCard 0.15 version https://github.com/d235j/360Controller
its signed

**Comment by electric-monk** [comment]
Yes, I need to update the version on my site with a new signature.

I assume a less useful alternative is to allow apps from "Anywhere" in the security settings until I get that done.

**Comment by jeffomatic** [comment]
I don't think the "Anywhere" setting applies to kernel extensions...I tried changing this and realized it was already set to "Anywhere." I may try compiling from source to see what happens.

**Comment by jjonesnook** [comment]
The beta 0.15 driver from rodrigocard worked for me first time. Thanks for the pointer.

Sent from my iPhone

On May 16, 2015, at 7:03 PM, Jeff Lee <notifications@github.com<mailto:notifications@github.com>> wrote:

I don't think the "Anywhere" setting applies to kernel extensions...I tried changing this and realized it was already set to "Anywhere." I may try compiling from source to see what happens.

##

**Comment by mele113** [comment]
Your welcome jjonesnook

**Comment by jaraco** [comment]
In case it's of any value, I've merged the d235j code with the electric-monk code in the jaraco fork. @electric-monk, Feel free to pull those changes if that saves you any time making a new release. I can make a Pull Request also if you prefer.
